### ISSUE TITLE
[Bug]: "Search Failed" and "Source Fetch Failed"

### ISSUE THREAD

**Issue by tskmri** [ThaUnknown/miru#534]
### Preflight checklist

- [X] I have searched the [issue tracker](https://github.com/ThaUnknown/miru/issues) for a bug report that matches the one I want to file, without success.
- [X] I have searched the [frequently asked questions](https://github.com/ThaUnknown/miru/blob/master/docs/faq.md) for a solution that fixes this problem, without success.
- [X] I have checked that I'm using the [latest stable](https://github.com/ThaUnknown/miru/releases/latest) version of the app.

### What app version are you using?

5.5.6

### What operating system version are you using?

Windows

### Expected Behavior

Normally when I click on anime to watch it'll load a list of torrent for me to choose from.

### Actual Behavior

I just get an error, "Source Fetch Failed", and "No torrent found for [Anime]". This happens with every single anime. Home feed also doesn't load for me always saying "Search Failed".

### Screenshots

![image](https://github.com/user-attachments/assets/47a04a07-1abe-4da9-838b-1619a2343bbb)
![image](https://github.com/user-attachments/assets/808014ce-05c5-4842-b57e-d0c5aa9b0a09)

**Comment by ThaUnknown** [comment]
this implies some network issue, I'm not sure what it could be, but it's definitely a network issue on your end

**Comment by nekwo** [comment]
anime tosho is down

**Comment by bee-san** [comment]
I am also getting this. Animetosho and seadex both appear to be up for me

**Comment by bee-san** [comment]
> 19th Sep 2024: Maintenance on the secondary webserver is planned to occur on 2024-09-21 06:00 GMT (affects attachments, screenshots, torrents, NZBs and feed). Downtime is expected to be around 1 hour, should no complications occur. UPDATE: server is currently experiencing boot failures, remaining downtime unknown.

Ah. maintenance day :)

**Comment by Eichen97** [comment]
Given this is the earliest psot, im linking my feature rquest here: #539

And the onther one somebody did without checking before, bc at least is more detailed #534

**Comment by blackwing12344** [comment]
> > 19th Sep 2024: Maintenance on the secondary webserver is planned to occur on 2024-09-21 06:00 GMT (affects attachments, screenshots, torrents, NZBs and feed). Downtime is expected to be around 1 hour, should no complications occur. UPDATE: server is currently experiencing boot failures, remaining downtime unknown.
> 
> Ah. maintenance day :)

where is this?

**Comment by Eichen97** [comment]
> where is this?

https://feed.animetosho.org

this is the rss feed that the whole app hangs on.

**Comment by blackwing12344** [comment]
Yup, links 100% dead. Is it possible to use a different feed?

**Comment by nekwo** [comment]
![image](https://github.com/user-attachments/assets/aaccb881-797a-46fa-9f96-0c27d4bba41b)

yea it's been down for over a day now

**Comment by Onigucci** [comment]
> this implies some network issue, I'm not sure what it could be, but it's definitely a network issue on your end

definitely u say

**Comment by ThaUnknown** [comment]
> > this implies some network issue, I'm not sure what it could be, but it's definitely a network issue on your end
> 
> definitely u say

yes, tosho has been down for less than 20 hours, this post was made over a week ago

**Comment by Onigucci** [comment]
> > > this implies some network issue, I'm not sure what it could be, but it's definitely a network issue on your end
> > 
> > 
> > definitely u say
> 
> yes, tosho has been down for less than 20 hours, this post was made over a week ago

fair, but support always quick to blame the other persons end instead of looking into things. Thought this was another one of those times. Apologies. 

This is my 2nd night of it not working. Do you know how much longer the maint is?

**Comment by ThaUnknown** [comment]
it's not called blame, it's called reading, he provided enough information for me to know it's a network issue, I know how the app I wrote myself works 

this is miru, not animetosho I recommend you ask animetosho

**Comment by Onigucci** [comment]
> it's not called blame, it's called reading, he provided enough information for me to know it's a network issue, I know how the app I wrote myself works
> 
> this is miru, not animetosho I recommend you ask animetosho
It looks like he shared what everyone is currently experiencing from what I can tell.

Here's my personal ss
![Screenshot 2024-09-22 0334003](https://github.com/user-attachments/assets/487b5edc-4874-4e12-b340-e25caf858d68)

And apologies again. Figured you would be up2date on issues connected to your app and might have the answer.

**Comment by nekwo** [comment]
it's been like 2 days, it's still down is there a way to program in a fallback url?????
